src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -99,7 +99,7 @@ private MobileAdsDataProvider() {
                 "\"ab\":[",
                 "\"dmn\":\"",
                 "\"skin\":\"mercury\"",
-                "\"ar\":\"4:3\"",
+                "\"ar\":\"3:4\"",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"ar\":\"3:4\"" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)